src/modules/ie-objects/controllers/ie-objects.controller.ts

```diff
@@ -170,12 +172,13 @@ export class IeObjectsController {
 		return convertObjectToCsv(objectMetadata);
 	}
 
-	@Get(':schemaIdentifier/related/:meemooIdentifier')
+	@Get(':schemaIdentifier/related/:meemooIdentifier/')
```

Comment:
Now that we figured out what the which endpoint is which, lets add some documentation to the swagger file:
```
	@ApiOperation({
		description:
			'Get objects that belong to the same meemoo identifier group. For instance objects that are highlights of the same soccer match.',
	})
```